railties/test/secrets_test.rb

```diff
@@ -158,7 +155,7 @@ def teardown
         assert_equal(secrets.dup.force_encoding(Encoding::ASCII_8BIT), IO.binread(tmp_path))
       end
 
-      assert_equal "00112233445566778899aabbccddeeff…\n", rails("runner", "-e", "production", "puts Rails.application.secrets.api_key", fork: false)
+      assert_equal "00112233445566778899aabbccddeeff…\n", rails("runner", "-e", "production", "puts Rails.application.secrets.api_key")
```

Comment:
We should also remove the `:fork` keyword argument from the `rails` method that @matthewd added.